controller_report_xls/controllers/main.py

```diff
@@ -38,6 +38,86 @@
 _logger = logging.getLogger(__name__)
 
 
+def is_float(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_integer(value):
+    try:
+        int(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_formatted_number(value, thousands_sep, decimal_point):
+    set_sign = set([thousands_sep, decimal_point, '-'])
+    set_val = set(list(value))
+    if set_val == set():
+        return False
+    elif any([val.isalpha() for val in set_val]):
```

Comment:
What about `s/any([val.isalpha() for val in set_val]):/any([val.isalpha() or val.isspace() for val in set_val]):/g`?